src/pytest_inmanta_lsm/remote_orchestrator.py

```diff
@@ -45,6 +45,7 @@ def __init__(
         settings: Dict[str, Union[bool, str, int]],
         noclean: bool,
         ssh_port: str = "22",
+        container_env: bool = False,
```

Comment:
It would be safer to add this new argument at the end of the argument list. Adding it here breaks the API of the `RemoteOrchestrator` class when an instance is created using positional arguments. I don't know whether this is an issue, as I don't know whether this class is used in another library as well, but I don't see any reason why we would break the API if we don't have to.

Reply:
Good point